renovate.json5

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended",
+    ":label(renovate)", // Add label.
+    ":prConcurrentLimit10", // Limit to maximum 10 open PRs.
+    ":timezone(Asia/Tokyo)",
+    ":enableVulnerabilityAlertsWithLabel(security)", // Raise PR when vulnerability alerts are detected with label security.
+    ":semanticCommitTypeAll(chore)", // If semantic commits detected, use semantic commit type chore for all
+  ],
+  "dependencyDashboardLabels": ["renovate"],
+  "schedule": ["after 3am on the first day of the month"], // Monthly(before 3am on the first day of the month) is unstable.
+  "packageRules": [
+    // The lower rule always takes precedence.
+    {
+      "groupName": "minor dependencies for npm",
+      "matchManagers": ["npm"],
+      "matchDepTypes": ["dependencies"],
+      "matchUpdateTypes": ["minor"],
+    },
```

Comment:
![must-badge](https://img.shields.io/badge/review-must-red.svg)
`patch` is not considered.